### ISSUE TITLE
Zoom mode in High Resolution mode seems buggy in some resolutions

### ISSUE THREAD

**Issue by Gagert** [libretro/libretro-uae#148]
In the game Lemmings 1, some items of the main menu appears sharper than in the more common Amiga screen resolution, because the game utilizes a higher resolution mode on some objects. However, while using Zoom mode it seems like the results differ if I either enable or disable High Resolution mode in Options.

With High Resolution mode off, setting Vertical Centering to none, Vertical Position to 10 and Zoom mode to larger, I can fill almost all of my screen in Lemmings 1 (with the correct pixel aspect ratio on a 1080p screen).

Enabling High Resolution mode however with the very same settings shrinks the image quite a bit. I can turn the Zoom mode slider up a notch to maximum, but it does not seem to affect the actual size of the image, other than moving the image down, making a chunk of the vertical pixels get eaten by a letterbox border in the bottom (not entirely reaching the lower part of my screen before getting inky black).

**Comment by sonninnos** [comment]
I don't understand the problem. Why not just use Vertical position: Automatic, since it works fine with zoom in this and pretty much every game..?

**Comment by rsn8887** [comment]
Are you sure you are using the correct settings stated underneath the zoom mode option? (Aspect Ratio: Core Provided, Integer Scaling: OFF etc...)

**Comment by sonninnos** [comment]
![retroarch_2019_10_12_04_15_07_654](https://user-images.githubusercontent.com/45124675/66692447-1525c080-eca7-11e9-90a2-fbfedf602551.png)

![retroarch_2019_10_12_04_16_00_079](https://user-images.githubusercontent.com/45124675/66692448-18b94780-eca7-11e9-9c04-c4e6b788edbc.png)

Both with maximum zoom and position automatic. Don't see a problem, except lores looks like crap.

**Comment by rsn8887** [comment]
As it should, because Lemmings is a hi-res game (technically med-res in Amiga lingo) :)

**Comment by Gagert** [comment]
My mistake; I had Integer Scaling set to On in RetroArch. Disabling Integer Scaling made everything look as it should.

The reason I usually have Integer Scaling on in RetroArch, is that some libretro cores tend to stretch the image the "wrong" way, by not making all pixels even squares, and making some of them appear as rectangles.

Thanks for the clarification; I'm amazed how far this libretro corehas come in terms of development, compared to when I first tried it (I tried running it without any success a couple of years ago).
